pkg/cluster/audit/audit.go

```diff
@@ -207,3 +208,104 @@ func decodeAuditID(auditID string) (time.Time, error) {
 	t := time.Unix(ts, 0)
 	return t, nil
 }
+
+type deleteAuditLog struct {
+	Files         []string  `json:"files"`
+	Size          int64     `json:"size"`
+	Count         int       `json:"count"`
+	DelBeforeTime time.Time `json:"delete_before_time"` // audit logs before `DelBeforeTime` will be deleted
+}
+
+// DeleteAuditLog  cleanup audit log
+func DeleteAuditLog(dir string, retainDay int, skipConfirm bool, displayMode string) error {
+	if retainDay < 0 {
+		return errors.Errorf("retain-time cannot be less than 0")
```

Comment:
should be match the cli argument

Reply:
It is to detect whether the input parameters are legal. Cli also needs to add this check.